flink-libraries/flink-gelly-examples/src/main/java/org/apache/flink/graph/drivers/input/InputBase.java

```diff
@@ -32,6 +35,9 @@
 extends ParameterizedBase
 implements Input<K, VV, EV> {
 
+	protected LongParameter parallelism = new LongParameter(this, "__parallelism")
+		.setDefaultValue(PARALLELISM_DEFAULT);
```

Comment:
I know, it's been that way before, but what a `LongParameter` although only the `intValue()` of it is actually used?